### PR TITLE
exciting new developments etc

### PR DESCRIPTION
**Pull request by rainbreak**

### CODE REVIEW

File: src/dss.md [hunk 1]
```diff
@@ -3098,13 +3098,13 @@ for all
...
     vat     |-> Vat
-    flapper |-> Flap
+    flapper |-> Flapper
     bump    |-> Bump
     hump    |-> Hump
     Sin     |-> Sin
     Ash     |-> Ash
 
-storage Flap
+storage Flapper
```
Comment (MrChico) [inline src/dss.md:3771]: This is good :+1:

File: src/dss.md [hunk 2]
```diff
@@ -4812,7 +4812,7 @@ iff in range uint256
 
 if
 
-    usr =/= ACCT_ID
```
Comment (mhhf) [inline src/dss.md]: desperation?

File: src/dss.md [hunk 3]
```diff
@@ -834,7 +834,7 @@ iff in range uint256
     Debt  + (Ilk_rate * dart)
     (Urn_art + dart) * Ilk_rate
     (Urn_ink + dink) * Ilk_spot
-    (Ilk_Art + dart) * Ilk_rate
+    (Ilk_art + dart) * Ilk_rate
```
Comment (rainbreak) [inline src/dss.md]: fake news

File: src/dirty_lemmas.k.md [hunk 4]
```diff
@@ -100,4 +100,11 @@ syntax IntList ::= bytesToWords ( WordStack )       [function]
 rule chop(A +Int B) >Int A => (A +Int B <=Int maxUInt256)
   requires #rangeUInt(256, A)
   andBool #rangeUInt(256, B)
+
+rule X *Int Y <=Int maxUInt256 => true
+  requires chop(chop(X *Int Y) /Int Y) ==Int X
+
+rule 0 <= Int X *Int Y => true
+  requires rangeUInt(256, X)
+  andBool rangeUInt(256, Y)
```
Comment (rainbreak) [inline src/dirty_lemmas.k.md]: should be useful for flopper_dent_fail_rough

**Comment by rainbreak** [comment]
latest run of this: https://dapp.ci/k-dss/4296d527e38631f8b756/

796/812 accepted proofs.

Remaining problem areas:

- vat lemmas for fold/grab/frob/fork fail cases
- skip/bite pass cases
- end.cage k performance (oom currently)
- drip fail cases are entering rpow loop and timing out
- flapper.tend fail case timing out

**Pull request merged**